deployment/build-s3-dist.sh

```diff
@@ -268,6 +268,9 @@ fi
 # Otherwise, chalice will use the existing deployment package
 [ -e .chalice/deployments ] && rm -rf .chalice/deployments
 
+echo "cp -R $source_dir/share ./dist"
+cp -R "$source_dir/share" "./dist"
```

Comment:
It would be good to use a more descriptive name than "share". What do you think about naming that directory "sigv4/"?